conference/react-js/src/components/MyInfo/MyInfo.jsx

```diff
@@ -0,0 +1,244 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import axios from 'axios';
+import { myInfo } from '../../constants/backend-url';
+
+import InputBlock from '../InputBlock/InputBlock';
+import TextareaBlock from '../TextareaBlock/index';
+import PopUpSaved from './PopUps/PopUpSaved';
+import PopUpPreventUnsavedExit from './PopUps/PopUpPreventUnsavedExit';
+import PopUpChangePhoto from './PopUps/PopUpChangePhoto';
+
+class MyInfo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showPreventUnsavedExitModal: false,
+      showInfoSavedModal: false,
+      showChangePhotoModal: false,
+      user: {},
+    };
+  }
+
+  componentDidMount() {
+    this.setDefaultValues(this.props);
+    console.log(this.props.user);
+  }
+
+  setDefaultValues = ({ user }) => {
+    this.setState({ user });
+  };
+
+  handleOpenModal = (e) => {
+    e.preventDefault();
+    this.setState({ showInfoSavedModal: true });
+  };
+
+  handleCloseModal = (e) => {
+    e.preventDefault();
+    this.setState({ showInfoSavedModal: false });
+  };
+
+  handleCloseModal1 = (e) => {
+    e.preventDefault();
+    this.setState({ showChangePhotoModal: false });
+  };
+
+  handleInput = (e) => {
+    e.preventDefault();
+    const upUser = {
+      ...this.state.user,
+      [e.target.name]: e.target.value,
+    };
+    this.setState(() => ({ user: upUser }));
+  };
+
+  handleSaveInfo = (e) => {
+    e.preventDefault();
+
+    const saveUserInfo = ({ bio, job, company }) => {
+      const body = { bio, job, company };
+      return axios.post(myInfo, body);
+    };
+
+    saveUserInfo(this.state.user)
+      .then((res) => {
+        console.log(res);
```

Comment:
same. remove logging from final version

Reply:
backend part is not ready for this post request yet, so I need something to know if there is an error or not